scripts/mint-nft.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/bash
```

Comment:
Ideally it shouldn't be in bash at all, but I guess it's not gonna be used in production?

Reply:
Should only be used locally until we fully automate the minting process, but I'll remove this just in case